.github/workflows/ci.yml

```
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.9, "3.10", 3.11, 3.12, 3.13]
+        python-version: [3.9, "3.10", 3.11, 3.12, 3.13, 3.14-dev]
```

```suggestion
        python-version: [3.9, "3.10", 3.11, 3.12, 3.13, 3.14]
```

Since we are now using uv to setup python (I believe)